doc/user-guide/user-guide.rst

```diff
@@ -1,74 +1,147 @@
+Basically the cycle build is subdivided in two levels; cycle and revision.
 
-  .. figure:: /images/filename.ext
-   :name: fig-label
+Anytime a version of a core package changes a new cycle must be created.
+If what changed is the version of an individual package or dependency library, outside the core packages, a new revision is created.
+The build is then tagged accordingly;
 
-   Caption text.
+.. prompt:: bash
 
-  Feel free to delete this instructional comment.
+  c<cycle-number>.<revision>
 
-.. Fill out data so contacts section below is auto-populated
-.. add name and email between the *'s below e.g. *Marie Smith <msmith@lsst.org>*
-.. |CSC_developer| replace::  *Replace-with-name-and-email*
-.. |CSC_product_owner| replace:: *Replace-with-name-and-email*
+All the images are tagged accordingly.
 
-.. _User_Guide:
+For instance, the image for the ``MTMount`` component for cycle 14, revision 3, will receive the name:
 
-#######################
-Build Cycle User Guide
-#######################
+.. prompt:: bash
 
-.. Update links and labels below
-.. image:: https://img.shields.io/badge/GitHub-ts_athexapod-green.svg
-    :target: https://github.com/lsst-ts/ts_athexapod
-.. image:: https://img.shields.io/badge/Jenkins-ts_athexapod-green.svg
-    :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_athexapod/
-.. image:: https://img.shields.io/badge/Jira-ts_athexapod-green.svg
-    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_athexapod
-.. image:: https://img.shields.io/badge/ts_xml-ATHexapod-green.svg
-    :target: https://ts-xml.lsst.io/sal_interfaces/ATHexapod.html
+  ts-dockerhub.lsst.org/mtmount:c0014.003
+
+The first part of the name is the nexus docker registry where we store the deployable artifacts, the second is the name of the image followed by the tag.
+
+In addition, for each cycle the latest revision is tagged with the cycle only, e.g.;
+
+.. prompt:: bash
+
+  ts-dockerhub.lsst.org/mtmount:c0014
+
+This allows for easily switching back and forth between revisions of a single cycle.
+At the same time, all images are annotated with the version information about the installed packages so one can easily inspect the content by doing:
+
+.. prompt:: bash
+
+  docker image inspect -f {{.Config.Labels}} ts-dockerhub.lsst.org/mtmount:c<cycle-number>.<revision>
+
+.. _Core-packages:
+
+Core packages
+-------------
+
+  - ts_xml: Contains the definition of the interfaces of all components in the system.
+  - ts_sal: Contains the core DDS communication middleware layer.
+  - ts_idl: Contains ancillary files used by ts_salobj and other higher level applications.
+  - ts_salobj: High-level Python library to develop CSCs and SAL Script.
+
+
+.. _Building-a-new-Cycle-or-Revision:
+
+Building a new Cycle or Revision
+================================
+
+There are two ways users can run a build; manually on a private server or using Jenkins server.
```

Comment:
replace "Jenkins server" with "the TSSW Jenkins server". Add a link to the Jenkins page too.